src/lib/pages/backoffice/Loan/LoanDetails/LoanOverbooked/LoanOverbooked.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { Segment } from 'semantic-ui-react';
+import { documentApi } from '@api/documents';
+
+export default class LoanOverbooked extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      overbooked: false,
+    };
+  }
+
+
+  componentDidMount() {
+    this.fetchCirculation()
+  }
+
+  componentWillUnmount() {
+    this.cancellableCirculation && this.cancellableCirculation.cancel();
+  }
+
+  fetchCirculation = async () => {
+    const { documentPid } = this.props;
+    try {
+      console.log(documentPid);
+      this.cancellableCirculation = withCancel(
+        documentApi.get(documentPid)
+      );
+      const response = await this.cancellableCirculation.promise;
+      const circulation = response.data.metadata.circulation;
+      this.setState({ overbooked: circulation.overbooked});
+    } catch (error) {
+      console.log(error)
```

Comment:
Missing the part when unmounting the component? Double check the other existing examples.

Reply:
I'm not so sure if the idea would be to just hide the warning in case the component is unmounted or the error should be displayed.
Also, then I'm no so sure why in src/lib/modules/Document/DocumentStats.js there is a comment instead of a proper handling of the error.

Reply:
> I'm not so sure if the idea would be to just hide the warning in case the component is unmounted or the error should be displayed.
When the component is `unmounted`, you cannot use `this.`, e.g. `this.setState`. We have discussed about this, you can read more for example [here](https://www.freecodecamp.org/news/how-to-work-with-react-the-right-way-to-avoid-some-common-pitfalls-fc9eb5e34d9e/).

> Also, then I'm no so sure why in src/lib/modules/Document/DocumentStats.js there is a comment instead of a proper handling of the error.
This is because in the case of document stats, we don't want to do anything at all. Imagine that you have an issue with the stats, you probably don't want to display an error message, a popup warning or banner to the user, on each book details page.